var/spack/packages/py-pandas/package.py

```diff
@@ -16,6 +16,10 @@ class PyPandas(Package):
     depends_on('py-scipy')
     depends_on('py-setuptools')
     depends_on('py-pytz')
+    depends_on('libdrm')
+    depends_on('libpciaccess')
+    depends_on('llvm')
```

Comment:
@alfredo-gimenez: is llvm necessary?  Is it just a build dep?  I may need to fast-track the build dependency feature...